### PR TITLE
siddhi-io-rabbitmq components

### PR DESCRIPTION
**Pull request by sivaramya**
siddhi-io-rabbitmq publish and receive siddhi events through rabbitmq broker.
The user story can be seen -
RabbitMQ sink -https://redmine.wso2.com/issues/6475
RabbitMQ source - https://redmine.wso2.com/issues/6475

### CODE REVIEW

File: component/pom.xml [hunk 1]
```diff
@@ -1,203 +1,158 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```
Comment (ramindu90) [inline component/pom.xml]: Any reason for removing the license header?

File: component/pom.xml [hunk 2]
```diff
@@ -1,203 +1,158 @@
...
-                            org.wso2.carbon.event.input.adapter.core.*,
-                            !javax.xml.namespace,
-                            javax.xml.namespace; version=0.0.0,
-                            *;resolution:=optional,
+                            *;resolution:=optional
                         </Import-Package>
+                        <Include-Resource>
+                            META-INF=target/classes/META-INF
+                        </Include-Resource>
                         <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (ramindu90) [inline component/pom.xml]: Please do not use dynamic imports. Use exact imports

File: component/src/main/java/org/wso2/extension/siddhi/io/rabbitmq/source/RabbitMQConsumer.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                            lock.unlock();
+                        }
+                    }
+                    String message = new String(body, "UTF-8");
+                    sourceEventListener.onEvent(message, null);
+                } catch (IOException e) {
+                    log.error("Error in receiving the message from the RabbitMQ broker " +
+                            "in " + sourceEventListener, e);
+                }
+
```
Comment (ramindu90) [inline component/src/main/java/org/wso2/extension/siddhi/io/rabbitmq/source/RabbitMQConsumer.java]: Please remove these unnecessary spaces.

File: component/pom.xml (hunk 1 above)
Comment (sivaramya) [reply component/pom.xml]: fixed in  1f21b29

File: component/pom.xml (hunk 2 above)
Comment (sivaramya) [reply component/pom.xml]: fixed in  1f21b29

File: component/src/main/java/org/wso2/extension/siddhi/io/rabbitmq/source/RabbitMQConsumer.java (hunk 3 above)
Comment (sivaramya) [reply component/src/main/java/org/wso2/extension/siddhi/io/rabbitmq/source/RabbitMQConsumer.java]: fixed in  1f21b29

File: component/pom.xml [hunk 4]
```diff
@@ -1,203 +1,182 @@
...
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testing.version}</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.rabbitmq</groupId>
-            <artifactId>amqp-client</artifactId>
-            <version>${version.com.rabbitmq}</version>
+            <groupId>org.wso2.extension.siddhi.io.file</groupId>
```
Comment (ramindu90) [inline component/pom.xml]: Are you using this for testing purpose?

Comment (sivaramya) [reply component/pom.xml]: fixed in 0138d8f

**Review by ramindu90 — APPROVED** [review]

**Pull request merged**